aurora config issue was :
```
Marlin/src/HAL/HAL_STM32F1/../../inc/SanityCheck.h:1557:4: error: #error "LED_CONTROL_MENU requires BLINKM, RGB_LED, RGBW_LED, PCA9533, PCA9632, or NEOPIXEL_LED."
```

--- a/buildroot/share/tests/black_stm32f407ve-tests
+++ b/buildroot/share/tests/black_stm32f407ve-tests
@@ -7,9 +7,6 @@
 set -e
 
 use_example_configs STM32/Black_STM32F407VET6
-
-opt_set MOTHERBOARD BOARD_BLACK_STM32F407VE
-
 exec_test $1 $2 "Full-featured Sample Black STM32F407VET6 config"
 
 # cleanup

--- a/platformio.ini
+++ b/platformio.ini
@@ -504,7 +504,7 @@ board = BigTree_SKR_Pro
 extra_scripts = pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py
 build_flags = ${common.build_flags}
   -DUSBCON -DUSBD_USE_CDC -DUSBD_VID=0x0483 -DUSB_PRODUCT=\"STM32F407ZG\"
-  -DTARGET_STM32F4 -DSTM32F407_5ZX -DVECT_TAB_OFFSET=0x8000
+  -DTARGET_STM32F4 -DSTM32F407_5ZX -DVECT_TAB_OFFSET=0x8000 -DHAVE_HWSERIAL6
 lib_deps = ${common.lib_deps}
 lib_ignore = Adafruit NeoPixel, c1921b4, TMC26XStepper, SailfishLCD, SailfishRGB_LED, SlowSoftI2CMaster
 src_filter = ${common.default_src_filter} +<src/HAL/HAL_STM32>

